### ISSUE TITLE
Action log/event log

### ISSUE THREAD

**Issue by ghost** [AdguardTeam/AdGuardDNS#400]
Recently I was trying to work out when a few things on my dashboard were changed, realising I had no way to find out.

It would be really handy to have an action/event log for changes made to the account.

No other personalised DNS service I know of does this yet.

A few examples of things I'd like to see are:
- time/date stamps
- server name changes
- device name changes
- server add/remove
- device add/remove
- settings enabled/disabled
- successful/failed login attempts
- device connected
- service interruptions 

Basically anything that can be changed, the ultimate goal would be to log it.
